whisperv6/peer.go

```diff
@@ -208,19 +218,23 @@ func (peer *Peer) broadcast() error {
 	}
 
 	if len(bundle) > 0 {
-		// transmit the batch of envelopes
-		if err := p2p.Send(peer.ws, messagesCode, bundle); err != nil {
-			return err
+		batchHash, err := sendBundle(peer.ws, bundle)
```

Comment:
Why not something like this to avoid too many changes:
```go
batchHash := makeBatchHash(bundle)
if err := p2p.Send(peer.ws, messagesCode, bundle); err != nil {
         log.Warn("failed to deliver envelopes", "peer", peer.peer.ID(), "error", err)
	return err
}
```

Also, in the previous case, in case of an error, the function returned the error early. Now, it only logs a warning. Is this expected?

Reply:
sendBundle encodes list of envelopes only once. with a proposed change it will be encoded two times

Reply:
right, need to exit earlier